apps/prairielearn/src/pages/instructorAssessmentQuestions/components/EditQuestionModal.tsx

```diff
@@ -0,0 +1,469 @@
+import clsx from 'clsx';
+import { Modal } from 'react-bootstrap';
+
+import { useMemo, useState } from '@prairielearn/preact-cjs/hooks';
+
+import type { QuestionAlternativeJson, ZoneQuestionJson } from '../../../schemas/infoAssessment.js';
```

Comment:
Other editors disable the edit button in certain conditions, based on the `canEdit` flag. We need to do the same here. E.g. you cannot edit the example course.

<img width="1206" height="1150" alt="CleanShot 2025-11-24 at 10 40 25@2x" src="https://github.com/user-attachments/assets/4e5e5a1c-0a03-46fe-b0ff-09cc99cda788" />